src/Powercord/plugins/pc-moduleManager/components/parts/BaseProduct.jsx

```diff
@@ -43,19 +46,47 @@ class BaseProduct extends React.PureComponent {
     return (
       <>
         <Divider/>
-        <div className='powercord-product-footer'>
-          {this.props.product.discord && // @todo: i18n
-          <Tooltip text='Go to their Discord support server'>
-            <Clickable onClick={() => this.goToDiscord(this.props.product.discord)}>
-              <Discord/>
-            </Clickable>
-          </Tooltip>}
-          {typeof this.props.goToSettings === 'function' && // @todo: i18n
-          <Tooltip text='Settings'>
-            <Clickable onClick={() => this.props.goToSettings()}>
-              <Gear/>
-            </Clickable>
-          </Tooltip>}
+        <div className=' powercord-product-footer'>
+          {this.props.product.discord &&
+          <Button
+            onClick={() => this.goToDiscord(this.props.product.discord)}
+            look={Button.Looks.LINK}
+            size={Button.Sizes.SMALL}
+            color={Button.Colors.TRANSPARENT}
+          > {Messages.REPLUGGED_PLUGINS_DISCORD}
+          </Button>
+          }
+
+          {this.props.product.website &&
+              <Button
+                onClick={() => openExternal(this.props.product.website)}
+                look={Button.Looks.LINK}
+                size={Button.Sizes.SMALL}
+                color={Button.Colors.TRANSPARENT}
+              > {Messages.REPLUGGED_PLUGINS_WEBSITE}
+              </Button>
+          }
+
+          {
+            <Button
+              onClick={async () => openExternal(await this.getGitInfo(this.props.Path))}
+              look={Button.Looks.LINK}
+              size={Button.Sizes.SMALL}
+              color={Button.Colors.TRANSPARENT}
+            > {Messages.REPLUGGED_PLUGINS_GITHUB}
+            </Button>
+          }
+
+          {
+            <Button
+              onClick={() => openExternal(this.props.Path)}
```

Comment:
For some reason this is giving me `Error: Invalid URL` when I click on it. Finder does not open .

Reply:
I replace openExternal for openPath this should work hopefully
on commit [556b579](https://github.com/replugged-org/replugged/pull/41/commits/556b579d4364d44538f694a7d6627a7c57a5af74)